test/e2e-olm/elasticsearch_test.go

```diff
@@ -407,9 +407,7 @@ func rollingRestartTest(t *testing.T) {
 	memValue := resource.MustParse("1536Mi")
 	cpuValue := resource.MustParse("200m")
 	desiredResources := corev1.ResourceRequirements{
-		Limits: corev1.ResourceList{
-			corev1.ResourceMemory: memValue,
-		},
+		Limits: cr.Spec.Spec.Resources.Limits,
```

Comment:
i hope this doesn't fix it... if it does we have some issues in our comparisons

Reply:
It fixed it for me locally :fearful:

Reply:
i misread what this was doing... saw your comments in slack and it makes sense now